src/BugsnagUnity/SimpleJson.cs

```diff
@@ -465,7 +465,7 @@ public override IEnumerable<string> GetDynamicMemberNames()
     }
 }
 
-namespace SimpleJson
```

Comment:
What about `Bugsnag`or `BugsnagUnity` here, to be consistent with the rest of the package? It would also reduce any developer confusion in intellisense to start typing `Bugsnag…` and get `…SimpleJson`.

Reply:
Updated to use `BugsnagUnity` for consistency with the rest of the project, then retested against the repro case.